entc/integration/ent/card.go

```diff
@@ -115,13 +115,23 @@ func (c *Card) assignValues(columns []string, values []interface{}) error {
 			if value, ok := values[i].(*sql.NullTime); !ok {
 				return fmt.Errorf("unexpected type %T for field create_time", values[i])
 			} else if value.Valid {
-				c.CreateTime = value.Time
+				if c.config.readIntoTz != nil {
+					c.CreateTime = value.Time.In(c.config.readIntoTz)
+				} else {
+					c.CreateTime = value.Time
+				}
+
```

Comment:
could replace this with a `InReadIntoTz()` helper or something to keep it simple here and move the if into the `config` ?